servers/rendering/rasterizer_rd/shaders/scene_high_end.glsl

```diff
@@ -267,6 +261,14 @@ VERTEX_SHADER_CODE
 #else
 	gl_Position = projection_matrix * vec4(vertex_interp, 1.0);
 #endif
+
+#ifdef MODE_RENDER_DEPTH
+	if (scene_data.pancake_shadows) {
+		if (gl_Position.z <= 0.00001) {
```

Comment:
Shouldn't this be a variable amount based on the pancake size?

Reply:
no.. the pancake is applied to the orthogonal frustum basically by making it X distance larger towards the light from the slice sphere, so all you have to do is clamp all geometry to the beginning.